### ISSUE TITLE
Training stuck at epoch 1

### ISSUE THREAD

**Issue by Suppersine** [gantheory/TPA-LSTM#17]
I was deploying your model on COLAB - with the repo copied, I ran the code below:

!python main.py --mode train \
    --attention_len 16 \
    --batch_size 32 \
    --data_set muse \
    --dropout 0.2 \
    --learning_rate 1e-5 \
    --model_dir ./models/model \
    --num_epochs 40 \
    --num_layers 3 \
    --num_units 338

Everything seems to be fine at that point, but it's stuck while training - I don't know why.T_T
https://colab.research.google.com/drive/1jaHTWg637wkD35C5LC5rRjej_G95L6tV?usp=sharing

![2021-08-25 (7)](https://user-images.githubusercontent.com/49264553/130769287-69581ee4-59b6-4a6b-93aa-4237f3c7a46e.png)

**Comment by liuyue777** [comment]
> I was deploying your model on COLAB - with the repo copied, I ran the code below:
> 
> !python main.py --mode train --attention_len 16 --batch_size 32 --data_set muse --dropout 0.2 --learning_rate 1e-5 --model_dir ./models/model --num_epochs 40 --num_layers 3 --num_units 338
> 
> Everything seems to be fine at that point, but it's stuck while training - I don't know why.T_T https://colab.research.google.com/drive/1jaHTWg637wkD35C5LC5rRjej_G95L6tV?usp=sharing
> 
> ![2021-08-25 (7)](https://user-images.githubusercontent.com/49264553/130769287-69581ee4-59b6-4a6b-93aa-4237f3c7a46e.png)

Hi, I have the same problem, have you found a solution?
